index.html

```diff
@@ -1926,34 +1908,25 @@ <h2>Chunk layout</h2>
 <!-- ************Page Break******************* -->
 <!-- ************Page Break******************* -->
 <!-- Maintain a fragment named "5Chunk-naming-conventions" to preserve incoming links to it -->
-<section id="5Chunk-naming-conventions">
-<h2>Chunk naming
-conventions</h2>
+<section id="chunk-naming-conventions">
```

Comment:
Don't change the id of sections, to avoid breaking incoming links

Reply:
Incoming from where? I have `s/chunk-naming-conventions/5Chunk-naming-conventions/`

Reply:
Incoming from any other place on the web that links to it.